Week1/credit/credit.c

```diff
@@ -0,0 +1,83 @@
+#include <cs50.h>
+#include <stdio.h>
+
+int main(void)
+{
+
+    long creditcardnumber = get_long("Creditcardnumber: ");
+
+    int numberofdigits = 0;
+    long usedcreditcardnumber = creditcardnumber;
+    while (usedcreditcardnumber > 0)
+    {
+        usedcreditcardnumber = usedcreditcardnumber / 10;
+        numberofdigits++;
+    }
+
+    if (numberofdigits != 13 && numberofdigits != 15 && numberofdigits != 16)
+    {
+        printf("INVALID\n");
+
+        return 0;
+    }
+
+    int sum1 = 0;
+    int sum2 = 0;
+    long x = creditcardnumber;
```

Comment:
```suggestion
    long x = creditcardnumber;
```

Why transfer the variable into another? If you want a shorter variable nam, try to choose an abbreviation or initialism like `ccn` for `credit card number`.